docs/modules/storage.md

```diff
@@ -66,7 +67,8 @@ Updates an existing storage account.
 | ------------------------- | ------------------- | ------------------------------------------------------------ | -------- |
 | ` enableNonHttpsTraffic ` | `string`            | Specify whether non-https traffic is enabled. Allowed values:["enabled", "disabled"]. | N        |
 | `tags`                    | `map[string]string` | Tags to be applied to new resources, specified as key/value pairs. | N        |
-| `accessTier`              | `string`            | The access tier used for billing.    Allowed values: ["Hot", "Cool"]. Hot storage is optimized for storing data that is accessed frequently ,and cool storage is optimized for storing data that is infrequently accessed and stored for at least 30 days. **Note** : this field doesn't exist for plan `general-purpose-storage-account`. | N        |
+| `accessTier`              | `string`            | The access tier used for billing.    Allowed values: ["Hot", "Cool"]. Hot storage is optimized for storing data that is accessed frequently ,and cool storage is optimized for storing data that is infrequently accessed and stored for at least 30 days. **Note1** : this field doesn't exist for plan `general-purpose-storage-account`. **Note2** : this field can only set to "Hot" if you use "Premium_LRS" `accountType` | N        |
```

Comment:
What about "doesn't exist" -> "is invalid"? And note2 -> "Premium_LRS" account only supports "Hot" in this field?

Reply:
Good suggestion. I will change it.